src/openMVG/localization/LocalizationResult.hpp

```diff
@@ -122,11 +122,57 @@ class LocalizationResult {
     return _isValid;
   }
   
+  /**
+   * @brief Compute the residual for each 2D-3D association.
+   * @return A 2xN matrix containing the x-y residual for each point.
+   */
   Mat2X computeAllResiduals() const;
   
+  /**
+   * @brief Compute the residual for the inlier 2D-3D association.
+   * @return A 2xNumInliers containing the x-y residual for each inlier point.
+   */
   Mat2X computeInliersResiduals() const ;
+  
+  /**
+   * @brief Compute the reprojection error for the inliers.
+   * @return A 1xNumInliers vector containing the reprojection error for each inlier point.
+   */
+  Vec computeReprojectionErrorPerInlier() const;
+  
+   /**
+   * @brief Compute the reprojection error for the all the points.
+   * @return A 1xNumInliers vector containing the reprojection error for each point.
+   */
+  Vec computeReprojectionErrorPerAll() const;
+  
+  /**
+   * @brief Compute the RMSE for the inlier association.
+   * @return The RMSE for the inlier associations.
+   */
   double computeInliersRMSE() const ;
 
+  /**
+   * @brief Compute the RMSE for the all the associations.
+   * @return The RMSE for the inlier associations.
+   */
+  double computeAllRMSE() const ;
+  
+  /**
+   * @brief Re-compute the inliers according to the given threshold.
+   * @param[in] threshold The threshold for the reprojection error in pixels.
+   * @return The number of inliers detected with the new threshold.
+   */
+  std::size_t updateInliers(double threshold);
+  
+  /**
+   * @brief Re-compute the inliers using the threshold computed during resection.
```

Comment:
Select the best inliers according to the reprojection error threshold used/computed during the resection.